pkg/k8s/service.go

```diff
@@ -142,13 +143,52 @@ func ParseService(svc *types.Service) (ServiceID, *Service) {
 		}
 	}
 
-	return svcID, svcInfo
+	var svcInfoExternal []*Service
```

Comment:
Why can't you just extend the `Service` struct by adding a field to store external IPs? Then in the `pkg/k8s/watchers/service.go` create the ExternalIP services in similar way as we currently do for NodePort? This would allow us to get rid of a lot of new code below.

Reply:
Sorry @brb I know we have briefly discuss this offline but I don't see a good way to do it properly since an externalIP is in the end a frontend. I'll be happy to discuss this again and address it before 1.7 is released